gulpfile.js/tests/configGenerator.test.js

```diff
@@ -0,0 +1,123 @@
+var fs = require('fs')
+var test = require('tape')
+var configGenerator = require('./../util/backstop/configGenerator')
+var deepClone = require('./../util/deepClone')
+var testFixturesPath = './gulpfile.js/tests/fixtures/configGenerator/'
+var config = {
+  compLib: {
+    port: 7000,
+    host: 'http://example',
+    baseDir: testFixturesPath
+  },
+  vrt: {
+    backstopConfigTemplate: testFixturesPath + 'backstop.template.json',
+    backstopConfig: testFixturesPath + 'backstop.json'
+  }
+}
+
+var removeFiles = function (files) {
+  return new Promise(function (resolve) {
+    files.forEach(function (file) {
+      fs.unlink(file, resolve)
+    })
+  })
+}
+
+var getBackstopConfigData = function () {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(config.vrt.backstopConfig, function (err, data) {
```

Comment:
The above implementation with a `require` gives me an error because it is trying to get the file before it has been [created](https://github.com/hmrc/assets-frontend/pull/756/files#diff-af54faa9f889c64db197c872e67b4776R65) in the promise chain. I am presuming because of writing to disk.

`[Error: Cannot find module './gulpfile.js/tests/fixtures/configGenerator/backstop.json'] code: 'MODULE_NOT_FOUND'`